### PR TITLE
2041 Add retry for storing refreshed user tokens

### PR DESCRIPTION
**Pull request by zamorastp**

### CODE REVIEW

**Comment by zamorastp** [comment]
@nfrydenholm @mbanasiewicz @clausjoergensen

File: Sources/AccountSDKIOSWeb/Client/Client.swift [hunk 1]
```diff
@@ -212,20 +212,33 @@ public class Client: CustomStringConvertible {
...
+    
+    private func storeSession(userSession: UserSession, attempts: Int = 1, completion: @escaping (Result<UserTokens, RefreshTokenError>) -> Void) {
+        func retry(_ attempts: Int) {
+            self.sessionStorage.store(userSession, accessGroup: nil) { result in
+                switch result {
+                case .success():
+                    completion(.success(userSession.userTokens))
+                case .failure(let error):
+                    if attempts > 0 {
+                        SchibstedAccountLogger.instance.debug("Failed to store refreshed tokens. Trying again...")
```
Comment (nfrydenholm) [inline Sources/AccountSDKIOSWeb/Client/Client.swift:231]: Could/should this be logged as an Info?

**Review by nfrydenholm — APPROVED** [review]
Nice and simple retry solution. Let's try it.

**Comment by mbanasiewicz** [comment]
Can we have some tests for it? 👼

**Comment by zamorastp** [comment]
> Can we have some tests for it?

Sure. Added unit tests

**Review by clausjoergensen — APPROVED** [review]

**Pull request merged**